capnp/src/private/layout_test.rs

```diff
@@ -61,26 +61,26 @@ fn simple_raw_data_struct() {
         // TODO the rest of uints.
 
         // Bits.
-        assert_eq!(reader.get_bool_field(0), true);
-        assert_eq!(reader.get_bool_field(1), false);
-        assert_eq!(reader.get_bool_field(2), false);
-        assert_eq!(reader.get_bool_field(3), false);
-        assert_eq!(reader.get_bool_field(4), false);
-        assert_eq!(reader.get_bool_field(5), false);
-        assert_eq!(reader.get_bool_field(6), false);
-        assert_eq!(reader.get_bool_field(7), false);
-
-        assert_eq!(reader.get_bool_field(8),  true);
-        assert_eq!(reader.get_bool_field(9),  true);
-        assert_eq!(reader.get_bool_field(10), false);
-        assert_eq!(reader.get_bool_field(11), false);
-        assert_eq!(reader.get_bool_field(12), false);
-        assert_eq!(reader.get_bool_field(13), true);
-        assert_eq!(reader.get_bool_field(14), false);
-        assert_eq!(reader.get_bool_field(15), false);
+        assert!(reader.get_bool_field(0));
+        assert!(!reader.get_bool_field(1));
```

Comment:
The `assert_eq!()` version is much more clear for this.

Reply:
At a glance, it's easy to overlook the `!`.

Reply:
noted. I'll close this PR, and this lint can be suppressed